definitions/clamp-js_v0.7.x/flow_>=v0.23.x/clamp-js_v0.7.x.js

```diff
@@ -0,0 +1,12 @@
+type ClampOptions = {
+  clamp?: number|string;
+  useNativeClamp?: bool,
+  truncationChar?: string,
+  truncationHTML?: string,
+  splitOnChars?: Array<string>,
+  animate?: bool,
+}
+
+declare module 'Clamp-js' {
```

Comment:
Per npm: Looks like the module name in npm is all lowercase (doesn't matter on osx with case-insensitive fs, but does on linux and other fses)